alerts/cluster-network-operator/hierarchy/alerts-hierarchy.dot

```diff
@@ -15,7 +15,7 @@ rankdir=LR;
     OVNKubernetesSouthboundDatabaseInboundConnectionMissing -> OVNKubernetesSouthboundDatabaseInboundConnectionError
 
 
-    NoOvnMasterLeader -> NoRunningOvnMaster
+    NoOvnClusterManagerLeader -> NoRunningOvnControlPlane
     V4SubnetAllocationThresholdExceeded
     V6SubnetAllocationThresholdExceeded
     NoOvnMasterLeader
```

Comment:
you need to remove this

Reply:
you also need to remove northd inactive since its currently disabled alert

Reply:
please double check OVNKubernetesControllerDisconnectedSouthboundDatabase and NodeWithoutOVNKubeNodePodRunning. I see conflicting info. I see you removed the runbooks but i am unsure if the alerts are actually gone.